### PR TITLE
add more confluence options

### PR DESCRIPTION
**Pull request by dustin-decker**
Adds options to confluence for:
- opting into attachment scanning
- opting out of history scanning

### CODE REVIEW

File: proto/sources.proto [hunk 1]
```diff
@@ -97,6 +97,8 @@ message Confluence {
   bool insecure_skip_verify_tls = 6;
   repeated string spaces = 7;
   repeated string ignore_spaces = 8;
+  bool include_attachments = 9;
```
Comment (trufflesteeeve) [inline proto/sources.proto:100]: By default this will not include attachments. Do we want the default to include them or exclude them?

**Review by trufflesteeeve — APPROVED** [review]
Looks good, but I could also see the option being `ExcludeAttachments` so they get scanned by default.

File: proto/sources.proto (hunk 1 above)
Comment (dustin-decker) [reply proto/sources.proto:100]: This was intentional since it was a recent addition with major performance implications

Comment (dustin-decker) [reply proto/sources.proto:100]: We'll be updating the docs soon to document all of the available options which will help discovery

**Pull request merged**